src/main/scala/com.gu.contentapi.client/Parameters.scala

```diff
@@ -1,6 +1,6 @@
 package com.gu.contentapi.client
 
-import org.joda.time.ReadableInstant
```

Comment:
`ReadableInstant` is both implemented by `DateTime` and `Instant` in Joda, but the former is for human-readable times while the latter is for machine-readable times. I've opted for the latter here. (In Java 8, [they did not make that mistake](http://blog.joda.org/2009/11/why-jsr-310-isn-joda-time_4941.html))